.github/workflows/android-release-workflow.yml

```diff
@@ -151,12 +44,17 @@ jobs:
           sentry-auth-dsn: ${{ secrets.SENTRY_AUTH_DSN }}
           lokalise-key: ${{ secrets.LOKALISE_KEY }}
           lokalise-app-id: ${{ secrets.LOKALISE_APP_ID }}
-      - name: "Build release AAB artifact"
+      - name: "Build release AAB&APK artifacts"
         env:
           KEYSTORE_PASSWORD: ${{ secrets.KEYSTORE_PASSWORD }}
           KEY_ALIAS: ${{ secrets.KEY_ALIAS }}
           KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
-        run: ./gradlew bundleRelease
+        run: ./gradlew bundleRelease assembleRelease
+      - name: "Upload AAB to Firebase App Distribution"
```

Comment:
The release build that are published on Slack and Firebase won't work anymore, FYI

Reply:
how can we debug release builds on our local machines now?

Reply:
I know. QA's will sign via seed phrase

Reply:
You will debug by signing seed phrase as well